lib/systems/default.nix

```
@@ -82,6 +86,13 @@ let
       isCompatible = _: throw "2022-05-23: isCompatible has been removed in favor of canExecute, refer to the 22.11 changelog for details";
       # Derived meta-data
       useLLVM = final.isFreeBSD;
+      cpuModel = let
+        name = if args ? cpuModel then args.cpuModel else "generic";
```

```suggestion
        name = args.cpuModel or "generic";
```
can be simplified with `or`

---

Done